managers/restore-manager/RestoreTaskPoller.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
```

Comment:
I see that design of this poller is similar to BoshTaskPoller/UnlockResourcePoller
We should put some of the common methods in base class (BasePoller?) and let others extend it

Reply:
Yes!